### ISSUE TITLE
keys.py - figure out customization of symbol_key and punctuation lists

### ISSUE THREAD

**Issue by knausj85** [talonhub/community#1561]
After merging https://github.com/talonhub/community/pull/1555, we may want to sort out how to ease the customization of symbol_key and  punctuation list

- Currently, symbol_key_words is imported by create_spoken_forms.py. We may want to re-evaluate whether we really utilize this functionality so we can move to .talon-list. Otherwise, gotta explore other options
- `punctuation_words` (used to populate the punctuation list) is a subset of `symbol_key_words` (used to populate the symbol_key list) that's intended to be available in both command and dictation. `symbol_key_words` is only used in command mode.  It's unclear how we can cleanly port this to .talon-list files without duplication. Some options that come to mind...
     - Without the create_spoken_form dependency, we could simply update the existing symbol_key capture to utilize both lists, and port to .talon-list
     - a CSV with a custom parser where we can somehow specify the intended scope. 
     - simply split this out into its own python file for easier customization
     - insert other options here :)

**Comment by AndreasArvidsson** [comment]
I would probably just remove that feature from create spoken forms and instead just strip symbols. Once that is done we can just update captures to use both lists. No duplication needed.

**Comment by knausj85** [comment]
I'm inclined to agree with the approach of simply removing the symbols from create spoken forms. I'll mark this as to discuss for next time (hopefully)
